code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -8,6 +8,10 @@ emp_act
 
 */
 
+#define WEAK 1
```

Comment:
The order of these is reverse. You should also move these somewhere under `__DEFINES/`